gateway/src/main/java/com/shopapp/gateway/entity/Customer.kt

```diff
@@ -9,4 +12,37 @@ data class Customer(
     val isAcceptsMarketing: Boolean,
     val addressList: List<Address>,
     val defaultAddress: Address?
-)
\ No newline at end of file
+) : Parcelable {
```

Comment:
Try to use https://proandroiddev.com/parcelable-in-kotlin-here-comes-parcelize-b998d5a5fcac